static/js/config-providers.js

```diff
@@ -72,7 +72,7 @@ $(document).ready(function() { // eslint-disable-line max-lines
         var newData = [isDefault, [name, url, key, cat]];
         newznabProviders[id] = newData;
 
-        $('#editANewznabProvider').addOption(id, name);
+        $('#editANewznabProvider').prepend('<option value=' + id + '>' + name + '</option>');
         $('select#editANewznabProvider').prop('selectedIndex', 0);
```

Comment:
@Thraxis this is not working too: `$('select#editANewznabProvider').prop('selectedIndex', 0);` 

so no provider is selected after loading page